load.php

```diff
@@ -216,10 +216,14 @@ function perflab_render_generator() {
  *
  * @since 1.3.0
  *
- * @param string $module Slug of the module.
- * @return bool Whether the module can be loaded or not.
+ * @param string $module      Slug of the module.
+ * @param bool   $get_message Optional. Whether to return the message instead of a boolean. Default false.
+ *
+ * @return bool|string Whether the module can be loaded or not.
+ *                     If the module can't be loaded, it can return either `false`,
+ *                     or a string with the message to be shown to users when $get_message is `true`.
  */
-function perflab_can_load_module( $module ) {
+function perflab_can_load_module( $module, $get_message = false ) {
```

Comment:
Let's not overload this function; changing what a function does with a boolean parameter is not a good practice. We should either think about a clean way to implement this or just hardcode it somewhere in a way that it doesn't pollute any API and we can still improve it later.

If we want to work on an API as part of this PR, we should introduce a new function for this; the `can-load.php` file expected behavior could be extended, e.g. so that it could return either just the boolean (for BC) or an array with both the true/false result and a message.

But to not make this PR more complex than it already is, it may be more straightforward to simply hardcode the message into `admin/load.php` if the module is `database/sqlite`.

Reply:
Done 👍